specs/DevSpecs/TextRun.Underline.md

```diff
@@ -0,0 +1,95 @@
+# InputValidation
+
+## Proposal Link
+https://github.com/microsoft/AdaptiveCards/issues/3051
+
+## Dependencies
+No dependencies required
+
+## Rendering
+
+### UWP & .NET WPF
+
+TextBlock has a property called [TextDecorations](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.textblock.textdecorations) which include Underline
+
+### Android
+
+TextRuns are rendered using SpannableStrings, between the possible spans to use there is [UnderlineSpan](https://developer.android.com/reference/android/text/style/UnderlineSpan) 
+
+### iOS
+
+NSMutableAttributedString are able to set underlines in the whole text and in some specific ranges using [NSUnderlineStyle](https://developer.apple.com/documentation/uikit/nsunderlinestyle)
+
+### .NET HTML
+
+The \<u> tag allows underlining
+
+## Feature Defaults
+
+The underline property will have the <i>false</i> value by default signalizing that no underline is required
+
+## Backwards Compatibility Concerns
+
+Underline will be ignored if sent to an older version of the renderer. No breaking changes are introduced.
+
+## Fallback impact
+
+### 1.0 Fallback text
+None
+
+### 1.2 Element level fallback
+None
+
+## Warning and Error States
+None
+
+## Open Issues
+
+### Feature extensibility
+The property may include support multiple line underlining at some point which will make the property accept numerical values as well as boolean values
```

Comment:
If we aren't supporting it for now, let's remove it from the discussion. I think with Extensibility I actually meant if it would impact *Existing* Extensibility flow, and then second whether the feature itself was extensible in any way/shape/manner. Totally fair to say it's not and then let's not include it here.